src/Components/button.js

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+
 const Button = props => {
-  const { name, size, last } = props;
+  const { name, wide, color } = props;
   if (name) {
-    return <input className={`btn-0 size-${size} ${(last) ? 'last' : ''}`} type="button" value={name} />;
+    return <input className={`btn-0 size-${wide} ${(color) ? 'last' : ''}`} type="button" value={name} />;
   }
   return null;
 };
 
 Button.propTypes = {
   name: PropTypes.string.isRequired,
-  size: PropTypes.number.isRequired,
-  last: PropTypes.bool.isRequired,
+  wide: PropTypes.number.isRequired,
+  color: PropTypes.bool.isRequired,
 };
 export default Button;
```

Comment:
After you change these two points mentioned above, it will affect and automatically bring about changes inside `"ButtonPannel"`, because that's where these pros will be coming from